subprojects/language-native/src/integTest/groovy/org/gradle/language/cpp/CppCachingIntegrationTest.groovy

```diff
@@ -51,29 +52,116 @@ class CppCachingIntegrationTest extends AbstractCppInstalledToolChainIntegration
     }
 
     def 'compilation can be cached'() {
-        setupProjectInDirectory(temporaryFolder.testDirectory)
+        def f = new NativeCachingFixture(buildType: buildType)
```

Comment:
Let's not use single-letter variable names.

Reply:
I removed the test fixture.